hack/source-manifests.sh

```diff
@@ -40,10 +40,10 @@ function dump_noobaa_csv() {
 
 	echo "Dumping Noobaa csv using command: $IMAGE_RUN_CMD --entrypoint=/usr/local/bin/noobaa-operator $NOOBAA_IMAGE $noobaa_dump_csv_cmd"
 	# shellcheck disable=SC2086
-	($IMAGE_RUN_CMD --entrypoint=/usr/local/bin/noobaa-operator "$NOOBAA_IMAGE" $noobaa_dump_csv_cmd) > $NOOBAA_CSV
+	($IMAGE_RUN_CMD --platform="${GOOS}"/"${GOARCH}" --entrypoint=/usr/local/bin/noobaa-operator "$NOOBAA_IMAGE" $noobaa_dump_csv_cmd) > $NOOBAA_CSV
```

Comment:
Does noobaa push the image for both os and arch, If not, please hardcode this value so that it does not fail if someone tries for those arches where the image is not present